addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -665,7 +665,11 @@ private static AlertNode getAlertNode(String name, String desc, int risk, int co
         AlertNode instance = new AlertNode(0, name);
         instance.setUserObject(alert);
 
+        AlertNode instance2 = new AlertNode(1, name);
+        instance2.setUserObject(alert);
```

Comment:
Use a different alert with different other info.

Reply:
do you mean there should be two alerts with one instance each ?

Reply:
An alert is an instance, yes.

Reply:
I am still not clear. 
1. 
![Screenshot from 2023-02-22 10-28-35](https://user-images.githubusercontent.com/87857234/220526216-33f0b570-a928-42ba-9c09-f0d887f75ceb.png)


2. 
![Screenshot from 2023-02-22 10-27-31](https://user-images.githubusercontent.com/87857234/220526237-9e038638-1327-466c-8973-d610b41dfc33.png)

Please tell out of these which is correct.

Reply:
The second example.